src/DETHRACE/common/cutscene.c

```diff
@@ -59,21 +59,13 @@ void PlaySmackerFile(char* pSmack_name) {
     tPath_name the_path;
     br_colour* br_colours_ptr;
     tU8* smack_colours_ptr;
-    // Smack* smk;
```

Comment:
FYI, I was able to reverse the logic for carmageddon 2: https://github.com/madebr/rec2/blob/master/src/carma2/common/cutscene.c

Reply:
awesome. So much similarity. I'm sad they got rid of the `fuck_off` variable :P

Reply:
nevermind, I forgot you have to make up your own names. Well, probably what you have as `keyPressed` is really still `fuck_off` in the OG code! :)